app/views/users/two_factor_authentication_setup/index.html.erb

```diff
@@ -12,7 +12,7 @@
 
 <%= render PageHeadingComponent.new.with_content(@presenter.heading) %>
 
-<p class="mt-tiny"><%== @presenter.intro %></p>
+<p><%== @presenter.intro %></p>
```

Comment:
Again with the `<%==`

From:
https://github.com/18F/identity-idp/blob/69e9db5d4a683a38456ce4455fa0dd4d07109a18/app/presenters/two_factor_options_presenter.rb#L35-L43

and as far as I can tell, none of those 3 options have HTML in the copy

https://github.com/18F/identity-idp/blob/69e9db5d4a683a38456ce4455fa0dd4d07109a18/config/locales/two_factor_authentication/en.yml#L161-L163

https://github.com/18F/identity-idp/blob/69e9db5d4a683a38456ce4455fa0dd4d07109a18/config/locales/two_factor_authentication/en.yml#L125-L127

https://github.com/18F/identity-idp/blob/69e9db5d4a683a38456ce4455fa0dd4d07109a18/config/locales/two_factor_authentication/en.yml#L129

so I think we can "just" remove the `<%==` (fully aware [last time I suggested this I was wrong](https://github.com/18f/identity-idp/pull/5683#discussion_r764164110))

```suggestion
<p><%= @presenter.intro %></p>
```

Reply:
Makes sense!